admin/create-dumps.sh

```diff
@@ -89,6 +90,14 @@ else
     exit
 fi
 
+# Lock cron, so it cannot be accidentally terminated.
```

Comment:
We still have an additional lock file [listed in the crontab](https://github.com/metabrainz/listenbrainz-server/blob/master/docker/services/cron/crontab#L6). Do we want to keep both of them, or do you think that we can remove the crontab one and just keep this one?

Reply:
Keep both -- they serve very different functions. The crontab ones are actual exclusions that prevent multiple things from running at once and this PR is about a cooperative lock to prevent cron container termination at the wrong time.